### ISSUE TITLE
Login fails repeatedly

### ISSUE THREAD

**Issue by lgelauff** [commons-app/apps-android-commons#3246]
**Summary:** 

I keep getting'login failed' messages, despite being quite confident the password is correct.

_Before creating an issue, please search the existing issues to see if a similar one has already been created. You can search issues by specific labels (e.g. `label:nearby`) or just by typing keywords into the search filter._

**Steps to reproduce:** 

After downloading the Commons app, I tried to log in to it. I keep getting'login failed' messages, despite being quite confident the password is correct. It makes me suspect something else is off.

I have tried to log in both from a secure network and a mobile network, I normally get 2fa. I attached a screen shot. I already reset my password and that failed too. I have no trouble logging in through the browser.

**System logs:**

```
(logs sent via email?)

Need help? See https://github.com/commons-app/apps-android-commons/wiki/Getting-app-logs-from-Android-Studio
```

**Device and Android version:** 

API level: 28
Android version: 9
Device manufacturer: motorola
Device model: moto g(7)
Device: river
Network type: wifi
App version name: 2.11.0~93b948d20
User name: null 
**Commons app version:** 

2.11.0

**Screen-shots:** 

https://phabricator.wikimedia.org/F31409414
**Would you like to work on the issue?**

no

**Comment by GearGit** [comment]
Sometimes it might be the case that your internet connection is poor. Did you try the same in some other place with a good internet connection?

**Comment by lgelauff** [comment]
Yes, my last attempt was from an extremely well-connected network.

**Comment by misaochan** [comment]
Hi @lgelauff , sorry, we had issues with 2FA logins in 2.11.0. I've just released v2.12 to beta testing, which should contain the fix - it should be available in a few hours. Please let us know if it doesn't work for you. :)

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul now we don't face any problem with login so we can close this issue now
